tests/ciscotest.rb

```diff
@@ -225,6 +225,28 @@ def interface_cleanup(intf_name)
     config(*cfg)
   end
 
+  # setup fabricpath env if possible and populate the interfaces array
+  # otherwise cause a global skip
+  def fabricpath_testenv_setup
+    return unless node.product_id[/N7K/]
+    intf_array = Feature.compatible_interfaces('fabricpath')
+    vdc = Vdc.new(Vdc.default_vdc_name)
+    save_lr = vdc.limit_resource_module_type
+    if intf_array.empty? || save_lr != 'f2e f3'
```

Comment:
Instead of hard-coding 'f2e f3', use `FabricPath.supported_modules` ? (would require adding `self.supported_modules` to fabricpath)